packages/fireproof/test/database.test.js

```diff
@@ -313,3 +316,129 @@ describe('basic Database with subscription', function () {
     equals(didRun, 0)
   })
 })
+
+describe('database with files input', function () {
+  let db
+  let image1
+  let image2
+  let result
+
+  beforeEach(async function () {
+    await resetDirectory(MetaStore.dataDir, 'test-with-images')
+
+    db = new Database('test-with-images')
+
+    image1 = fs.readFileSync('/workspaces/fireproof/packages/fireproof/test/test-images/image1.jpg')
+    image2 = fs.readFileSync('/workspaces/fireproof/packages/fireproof/test/test-images/image2.jpg')
+    const doc = {
+      _id: 'images-main',
+      type: 'files',
+      _files: {
+        image1,
+        image2
+      }
+    }
+
+    result = await db.put(doc)
+  })
+
+  it('Should upload images', async function () {
+    // console.log(image1)
+
+    // console.log(result,"This is the result when the images are stored")
+
+    equals(result.id, 'images-main')
```

Comment:
they dont run in sequence, the environment is reset in beforeEach for all of them. or at least, I think that is safer